[2025-05-29T00:00:01.648+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cancellations_etl_dag.extract_data scheduled__2025-05-28T00:00:00+00:00 [queued]>
[2025-05-29T00:00:01.654+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cancellations_etl_dag.extract_data scheduled__2025-05-28T00:00:00+00:00 [queued]>
[2025-05-29T00:00:01.655+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-05-29T00:00:01.744+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): extract_data> on 2025-05-28 00:00:00+00:00
[2025-05-29T00:00:01.749+0000] {standard_task_runner.py:57} INFO - Started process 20810 to run task
[2025-05-29T00:00:01.752+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cancellations_etl_dag', 'extract_data', 'scheduled__2025-05-28T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/cancellations_etl_dag.py', '--cfg-path', '/tmp/tmptdx9o8ub']
[2025-05-29T00:00:01.756+0000] {standard_task_runner.py:85} INFO - Job 55: Subtask extract_data
[2025-05-29T00:00:01.816+0000] {task_command.py:415} INFO - Running <TaskInstance: cancellations_etl_dag.extract_data scheduled__2025-05-28T00:00:00+00:00 [running]> on host a5551f46ca35
[2025-05-29T00:00:01.888+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='cancellations_etl_dag' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T00:00:00+00:00'
[2025-05-29T00:00:01.890+0000] {logging_mixin.py:151} INFO - File data/raw/train_delays.csv already exists. Skipping download.
[2025-05-29T00:00:01.890+0000] {logging_mixin.py:151} INFO - File data/raw/cancellations.csv already exists. Skipping download.
[2025-05-29T00:00:01.891+0000] {logging_mixin.py:151} INFO - File data/raw/punctuality.csv already exists. Skipping download.
[2025-05-29T00:00:01.891+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-29T00:00:01.900+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=cancellations_etl_dag, task_id=extract_data, execution_date=20250528T000000, start_date=20250529T000001, end_date=20250529T000001
[2025-05-29T00:00:01.934+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-29T00:00:01.947+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
